kDrive/UI/Controller/MainTabViewController.swift

```diff
@@ -243,19 +250,66 @@ extension MainTabViewController: MainTabBarDelegate {
         floatingPanelViewController.trackAndObserve(scrollView: plusButtonFloatingPanel.tableView)
         present(floatingPanelViewController, animated: true)
     }
+
+    func avatarLongTouch() {
+        let viewControllers = viewControllers
+        guard let rootNavigationController = viewControllers?[safe: MainTabBarIndex.profile.rawValue] as? UINavigationController
+        else {
+            return
+        }
+
+        let generator = UIImpactFeedbackGenerator(style: .light)
+        generator.impactOccurred()
+
+        selectedIndex = MainTabBarIndex.profile.rawValue
+
+        router.presentAccountViewController(navigationController: rootNavigationController, animated: true)
+    }
+
+    func avatarDoubleTap() {
+        accountManager.switchToNextAvailableAccount()
+        guard let accountManager = accountManager.currentDriveFileManager else {
+            return
+        }
+
+        let generator = UIImpactFeedbackGenerator(style: .light)
+        generator.impactOccurred()
+
+        @InjectService var router: AppNavigable
```

Comment:
Do we need to import the router here as it's already a property of MainTabViewController?

Reply:
Good catch, that's a mistake.